BootloaderCorePkg/Tools/CommonUtility.py

```diff
@@ -360,6 +362,7 @@ def compress (in_file, alg, out_path = '', tool_dir = ''):
     compress_data = get_file_data(out_file)
     lz_hdr = LZ_HEADER ()
     lz_hdr.signature = sig.encode()
+    lz_hdr.svn = svn

```

Comment:
What is the purpose of this svn ?

Reply:
This is referenced for components inside a container. Currently, There are no empty fields available in component headers. Components are compressed using  LZ compress and and appended to component header.
Eg: CompressHdr = (LOADER_COMPRESSED_HEADER *) CompBuf

Same "LOADER_COMPRESSED_HEADER" would be used for SBL stages compression also.

Reply:
Got it. Thanks.